cms/static/js/spec/views/module_edit_spec.js

```diff
@@ -0,0 +1,264 @@
+(function() {
+    'use strict';
+    define(['jquery', 'common/js/components/utils/view_utils', 'js/spec_helpers/edit_helpers',
+            'js/views/module_edit', 'js/models/module_info', 'xmodule'],
+            function($, ViewUtils, edit_helpers, ModuleEdit, ModuleModel) {
+        return describe('ModuleEdit', function() {
```

Comment:
`describe` doesn't need to be returned (I think this is just the coffeescript implicit returns at work)

There's a few of these, line 49 below, 250, etc